code/game/jobs/job/science.dm

```diff
@@ -131,13 +131,13 @@
 	stat_modifiers = list(
 		STAT_MEC = 30,
 		STAT_COG = 25,
-		STAT_BIO = 25,
+		STAT_BIO = 30, // Enough to not flunk surgical operations on the chest for the sake of prosthetic implantation/replacement on fleshy patients. - Seb
 	)
 
-	perks = list(/datum/perk/surgical_master, /datum/perk/robotics_expert, /datum/perk/si_sci)
+	perks = list(/*/datum/perk/surgical_master,*/ /datum/perk/robotics_expert, /datum/perk/si_sci) // Roboticists don't need knowledge on ATK use for sealing wounds and fixing FLESH organs, which are the only two things this perk granted them. They deal with metal more than flesh. - Seb
```

Comment:
```suggestion
	perks = list(/datum/perk/surgical_master,/datum/perk/robotics_expert, /datum/perk/si_sci) // Robotics gives implants and augmentation for flesh and metal alike, they require the extra perk to help them step in for medical as well as per their SoP
```

Reply:
According to wiki, it says that Science taking over medical duties is currently not in use but if you say it does Lilly, I'll trust your judgement.